### PR TITLE
Update to latest 6.0 .Net Runtime/SDK

### PR DESCRIPTION
**Pull request by vbjay**
## Proposed changes

- Run  ```.\scripts\Update-DotnetVersion.ps1```
- Run ```git diff -U0 -w --no-color --ignore-blank-lines | git apply --cached --ignore-whitespace --unidiff-zero --``` to stage non whitespace changes.  This time I also staged whitespace on the global.json file to make it not have much of a diff in the future.  This prevents staging the blank line removal in the RepoLayout.props file that this script generates.

```
channel-version     : 6.0
latest-release      : 6.0.9
latest-release-date : 2022-09-13
latest-runtime      : 6.0.9
latest-sdk          : 6.0.401
support-phase       : lts
eol-date            : 2024-11-12
```

## Test methodology 

- Commit changes generated and see diff

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Review by RussKie — APPROVED** [review]

**Comment by RussKie** [comment]
Thank you

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-    "sdk": {
-      "version": "6.0.400",
-      "rollForward": "feature"
-    }
-}
+﻿{
+    "sdk":  {
+                "version":  "6.0.401",
+                "rollForward":  "feature"
+            }
```
[the remark this answers is not in the document]
Comment (vbjay) [reply global.json:5]: See https://stackoverflow.com/questions/33145377/how-to-change-tab-width-when-converting-to-json-in-powershell. I didn't want to add another dependancy like Newtonsoft module and the hack with string manipulation feels too hacky. The json is valid and frankly managed by script. Why I didn't do that.

**Comment by mstv** [comment]
Should we cherry-pick this to the release/4.0 branch, too?

**Comment by RussKie** [comment]
Done, thank you for the reminder.

**Comment by RussKie** [comment]
PowerShell handling of JSON is... imperfect.

**Pull request merged**